starport/templates/testutil/register.go

```diff
@@ -31,8 +35,16 @@ func Register(ctx *plush.Context, gen *genny.Generator, appPath string) error {
 		return fmt.Errorf("ctx is missing value for the key %s", modulePathKey)
 	}
 	path := filepath.Join(appPath, testUtilDir)
-	if _, err := os.Stat(path); !os.IsNotExist(err) {
-		return nil
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return gen.Box(stargateTemplate)
```

Comment:
Is `stargateTemplate` the whole `testutil` directory?
We should add a comment to precise this